<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="1" failed="1" ignored="0" total="3" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="1007" started-at="2020-06-20T15:14:13 EDT" finished-at="2020-06-20T15:14:14 EDT">
    <groups>
    </groups>
    <test name="Default test" duration-ms="1007" started-at="2020-06-20T15:14:13 EDT" finished-at="2020-06-20T15:14:14 EDT">
      <class name="testcases.testcase1">
        <test-method status="PASS" signature="setReport()[pri:0, instance:testcases.testcase1@1dfe2924]" name="setReport" is-config="true" duration-ms="247" started-at="2020-06-20T15:14:13 EDT" finished-at="2020-06-20T15:14:13 EDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setReport -->
        <test-method status="SKIP" signature="isSkip()[pri:0, instance:testcases.testcase1@1dfe2924]" name="isSkip" duration-ms="12" started-at="2020-06-20T15:14:13 EDT" finished-at="2020-06-20T15:14:13 EDT">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping the Test Case]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping the Test Case
at testcases.testcase1.isSkip(testcase1.java:67)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- isSkip -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult)[pri:0, instance:testcases.testcase1@1dfe2924]" name="tearDown" is-config="true" duration-ms="18" started-at="2020-06-20T15:14:13 EDT" finished-at="2020-06-20T15:14:13 EDT">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=isSkip status=SKIP method=testcase1.isSkip()[pri:0, instance:testcases.testcase1@1dfe2924] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="FAIL" signature="verifyAddPayee()[pri:0, instance:testcases.testcase1@1dfe2924]" name="verifyAddPayee" duration-ms="8" started-at="2020-06-20T15:14:13 EDT" finished-at="2020-06-20T15:14:13 EDT">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[did not expect to find [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: did not expect to find [true] but found [false]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.failNotEquals(Assert.java:969)
at org.testng.Assert.assertTrue(Assert.java:43)
at org.testng.Assert.assertTrue(Assert.java:53)
at testcases.testcase1.verifyAddPayee(testcase1.java:61)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyAddPayee -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult)[pri:0, instance:testcases.testcase1@1dfe2924]" name="tearDown" is-config="true" duration-ms="1" started-at="2020-06-20T15:14:13 EDT" finished-at="2020-06-20T15:14:13 EDT">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=verifyAddPayee status=FAILURE method=testcase1.verifyAddPayee()[pri:0, instance:testcases.testcase1@1dfe2924] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="verifyLogin()[pri:0, instance:testcases.testcase1@1dfe2924]" name="verifyLogin" duration-ms="2" started-at="2020-06-20T15:14:13 EDT" finished-at="2020-06-20T15:14:13 EDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyLogin -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult)[pri:0, instance:testcases.testcase1@1dfe2924]" name="tearDown" is-config="true" duration-ms="3" started-at="2020-06-20T15:14:13 EDT" finished-at="2020-06-20T15:14:13 EDT">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=verifyLogin status=SUCCESS method=testcase1.verifyLogin()[pri:0, instance:testcases.testcase1@1dfe2924] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="endReport()[pri:0, instance:testcases.testcase1@1dfe2924]" name="endReport" is-config="true" duration-ms="658" started-at="2020-06-20T15:14:13 EDT" finished-at="2020-06-20T15:14:14 EDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- endReport -->
      </class> <!-- testcases.testcase1 -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
